test/integration_serial_test.go

```diff
@@ -0,0 +1,209 @@
+package test
```

Comment:
NOTE: scroll to next comment in this file that marks which test is new. Everything inbetween are from `integration_test.go`